packages/cadl-ranch-specs/http/extensible-enums/main.cadl

```diff
@@ -6,7 +6,7 @@ using Cadl.Http;
 @scenarioService("/extensible-enums")
 namespace ExtensibleEnums;
```

Comment:
Think we should just change completely the structure of this test.

```cadl
namespace Enums
```

and then have `Extensible` and `Fixed` subnamespace showing the use case for both

Reply:
I made that change (I think) but I don't know if I did it right. The CI won't pass anyhow until cadl-azure-core is released. I wanted to get the extensible test in quicker so emitters could be updated. 

I also don't know if the fixed test makes any sense, but let me know what you think.